### DISCUSSION TITLE
How do you decide which projects to resurrect via fork? (Re: State Config saver)

### DISCUSSION THREAD

**Discussion by CCpt5** [wkpark/sd-webui-model-mixer#99] [General]
Hello wkpark!

I've noticed some of your work on A1111 extensions are via improving on older/abandonded projects (For example: udetailer). Do you pick what you do based on personal need, or do you perhaps take input?

There's an old A1111 extension that was super helpful for me (and I'm sure others) called "webui state" (Repo: https://github.com/ilian6806/stable-diffusion-webui-state/issues/53 and current status thread). 

It was a fairly simple but amazingly effective tool that automatically saved everything about your current TXT2IMG / IMG2IMG settings and would automatically restore your exact state if you closed A1111 and relaunched it. It also allowed you to export current settings and restore them later. So if you have a complicated method to upscale that includes many extensions w/ different settings, you could just save that and reopen it at any other point w/o having to set values from scratch.  
  
The extension stopped working prior to version 1.6, and the developer doesn't have much time to work on it. In the linked thread above he mentions he'll try to fix/update it, but had hoped for more contributions when originally coding/submitting the extension.

If you're ever seeking a new A1111 project to help maintain I'd hope you give that one a look :)  

Someone did attempt to create a new similar extension from scratch recently (available at the top of the extension list as "state manager", but it doesn't seem to operate in the same manner nor do I think it's working at all right now......

Either way have a fantastic New Year!

**Comment by wkpark** [comment]
Thank you for your information! and happy New Year to you too! 😁
